redbot/core/core_commands.py

```diff
@@ -1073,7 +1073,7 @@ class Core(commands.Cog, CoreLogic):
         else:
             await ctx.send(_("Done."))
 
-    @_set.command(name="game")
+    @_set.command(name="game", aliases=["playing"])
```

Comment:
Other commands use `-ing` as their main name, I think it would be good to do it here as well.
```suggestion
    @_set.command(name="playing", aliases=["game"])
```